src/Export/VTK.jl

```diff
@@ -283,6 +284,23 @@ function write_nodeset(vtk, grid::AbstractGrid, nodeset::String)
     return vtk
 end
 
+"""
+    write_facetset(vtk::VTKGridFile, grid::AbstractGrid, facetset::String)
```

Comment:
Would be nice to match the interface from `cellsets` allowing a vector/set of keys? 

Also, since this is not as trivial as cellsets, perhaps it could even be nice to have a lower-level interface providing just an `AbstractVecOrSet{<:FacetIndex}`, not requiring it to be stored in the grid?